### DISCUSSION TITLE
Spell Point don't work when the name is change

### DISCUSSION THREAD

**Discussion by SALEMAN4** [misthero/dnd5e-spellpoints#112] [Q&A]
Hello, sorry for the burden but when in the option of the mod I change the ressource name for "Mana" it doesn't work (like for level up change or just to consume point when spell is cast) but when I write again Spell Points, it works with no problem (I don't forget to change the name of the ressource on the sheet, when I change for "Mana" the purple ressource bar dissapear but come back when it's "Spell Points". Thank you in advance for your help.

![image_2024-06-25_112952986](https://github.com/misthero/dnd5e-spellpoints/assets/64664084/b5255f43-1fbd-4a96-aa6a-374f4209c26a)
![image_2024-06-25_113317353](https://github.com/misthero/dnd5e-spellpoints/assets/64664084/4e91c614-8089-46b6-bbe9-b693f74fbf7b)

Edit : I already tested it a new world with no other module and it still doesn't work. If the name is not precisely "Spell Points", it's not working.

**Comment by misthero** [comment]
You can just change the item name without changing the resource name in the configuration (that is left for backward compatibility.) 

If you want to change that in the configuration than the item won't work unless you change the "Custom Label" in the item:

![immagine](https://github.com/misthero/dnd5e-spellpoints/assets/3662610/04d7a63a-c11f-4368-8ebe-d83fe1fa0bb1)
